terraform/mod/k8s/master.sh

```diff
@@ -1,14 +1,15 @@
 #!/bin/bash
-set -e
+set -ex
 sudo kubeadm init --config /tmp/kubeadm_config.yaml \
   --ignore-preflight-errors=Swap,NumCPU,SystemVerification
 
 [ -d "$HOME"/.kube ] || mkdir -p "$HOME"/.kube
-ln -s /etc/kubernetes/admin.conf "$HOME"/.kube/config
+sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
+sudo chown $(id -u):$(id -g) $HOME/.kube/config
```

Comment:
running cp in interactive mode in automation script sounds like a bad idea.

Reply:
Fixed https://github.com/openebs/Mayastor/pull/268/commits/5b38dc66d2408d7b84cd517462429a5cd21232c9
TBH I just copied what kubeadm spit out.